app/performance_platform/rest.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+
+from flask import Blueprint, request, jsonify
+
+from app.dao.fact_notification_status_dao import get_total_notifications_for_date_range
+from app.dao.fact_processing_time_dao import get_processing_time_percentage_for_date_range
+from app.dao.services_dao import get_live_services_with_organisation
+from app.errors import register_errors
+from app.performance_platform.performance_platform_schema import performance_platform_request
+from app.schema_validation import validate
+
+performance_platform_blueprint = Blueprint('performance_platform', __name__, url_prefix='/performance-platform')
+
+register_errors(performance_platform_blueprint)
+
+
+@performance_platform_blueprint.route('')
+def get_performance_platform():
+    # All statistics are as of last night this matches the existing performance platform
+    # and avoids the need to query notifications.
+    if request.args:
+        # Is it ok to reuse this? - should probably create a new one
+        validate(request.args, performance_platform_request)
+
+    # If start and end date are not set, we are expecting today's stats.
+    today = str(datetime.utcnow().date())
+
+    start_date = datetime.strptime(request.args.get('start_date', today), '%Y-%m-%d').date()
+    end_date = datetime.strptime(request.args.get('end_date', today), '%Y-%m-%d').date()
+    results = get_total_notifications_for_date_range(start_date=start_date, end_date=end_date)
+    total_notifications, emails, sms, letters = transform_results_into_totals(results)
```

Comment:
Ah, there is one major difference here actually I think.

On the performance platform, the headline figures for how many emails, texts and letters we have sent are for ALL notifications over time I believe so from the start of Notify (or at least us recording the figure). Whereas, when we want to show say the last month of data for notifications sent per day, we only query for the last 30 days of data. At the moment, how this is written means our big totals at the top of the page would also only show the total number sent in the last 30 days which is probably not what we want.

Reply:
That's just a mistake of extra brackets

Reply:
Good spot - fixed